.github/dependatbot.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+updates:
+- package-ecosystem: 'npm'
+  directory: '/'
```

Comment:
I think you wanna check the `/functions/` directory and not the `/`, since it is the `package.json` inside `functions` that contains the external dependencies :)